dotCMS/src/main/java/com/dotcms/experiments/business/web/SelectedVariant.java

```diff
@@ -0,0 +1,41 @@
+package com.dotcms.experiments.business.web;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This is a {@link com.dotcms.variant.model.Variant selected to a User, this mean this user
+ * is going to be into the {@link com.dotcms.variant.model.Variant}'s {@link com.dotcms.experiments.model.Experiment}.
+ *
+ * @see ExperimentWebAPI#isUserIncluded(HttpServletRequest, HttpServletResponse)
+ * @see com.dotcms.variant.model.Variant
+ */
+public  class SelectedVariant implements Serializable {
+
+    /**
+     * {@link com.dotcms.variant.model.Variant}'s name
+     */
+    @JsonProperty("name")
+    private String name;
+
+    /**
+     * URL to render the {@link com.dotcms.experiments.model.Experiment}'s page in this {@link com.dotcms.variant.model.Variant}
+     */
+    @JsonProperty("url")
+    private String url;
+
+    public SelectedVariant(String name, String url) {
```

Comment:
same set to final and JsonCreator

Reply:
done https://github.com/dotCMS/core/pull/23334/commits/088b10e51e9d5864fb510bd1a7cc17241899e47e#diff-f14f138414a0e78f2a8cdaeaa998426d2b3c35639d67b8dcd420baa5f2714d5aR21